dropbox.sls

```diff
@@ -1,16 +1,21 @@
 # just 32-bit x86 installer available
+
+{% set SOURCE_PATH = 'https://clientupdates.dropboxstatic.com/dbx-releng/client/' %}
+{% set VERSION = '69.4.102' %}
```

Comment:
Please keep the old version available as well.

Reply:
I don't know if it makes sense to add an older versions, because dropbox also has an auto-update function and older clients are updated automatically anyway

Reply:
OK